projects/plugins/jetpack/extensions/blocks/subscriptions/edit.js

```diff
@@ -232,6 +234,11 @@ export function SubscriptionEdit( props ) {
 					successMessage={ successMessage }
 				/>
 			</InspectorControls>
+			<BlockControls>
+				<Toolbar>
+					<GetAddPaidPlanButton context={ 'toolbar' } />
```

Comment:
We may need to add back the `hasNewsletterPlans`  here or pass it to the component.

See https://github.com/Automattic/jetpack/pull/26417/files#diff-e0824d0e12a2d47e8a9baa2bc0533543147d25dfe276b1e0b09fee1b0df1675e

Reply:
Fixed in https://github.com/Automattic/jetpack/pull/26417/commits/65e6f12142c9fe4aa51a9680b7d630ab0c88bf05